amundsen_application/static/js/components/DashboardPage/QueryListItem/index.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import { OverlayTrigger, Popover } from 'react-bootstrap';
+
+import "./styles.scss";
+
+export interface QueryListItemProps {
+  text: string;
+  url: string;
+  name: string;
+}
+
+type GoToDashboardLinkProps = {
+  url: string;
+};
+
+const QUERY_LABEL = "Query";
+const MODE_LINK_TOOLTIP_TEXT = "View in Mode";
+const LOADING_QUERY_MESSAGE = "Loading Query Component, please wait...";
+
+const LazyComponent = React.lazy(() => import("./CodeBlock"));
+
+const GoToDashboardLink = ({ url }: GoToDashboardLinkProps) => {
+  const popoverHoverFocus = (<Popover id="popover-trigger-hover-focus">{MODE_LINK_TOOLTIP_TEXT}</Popover>);
+
+  return (
+    <OverlayTrigger
+      trigger={["hover", "focus"]}
+      placement="bottom"
```

Comment:
Very minor, but what do you think of updating this to `top` and proposing that to design? I know our designs have this tooltip at the bottom, but I'm used to the concept of placing tooltips where they stand to obscure the least amount of information, and the top placement will extend into the whitespace above the component vs into the item itself.

Reply:
That makes sense, I will suggest it to Knowl!

Reply:
Accepted and updated!